src/person/tasks.py

```diff
@@ -35,8 +37,24 @@ def update_paused_users():
     every=12,
     period=IntervalSchedule.HOURS,
 )
+
 PeriodicTask.objects.get_or_create(
     interval=schedule,
     name="Pause Pear Updater",
     task="person.tasks.update_paused_users",
 )
+
+
+@shared_task
+def update_inactive_users():
```

Comment:
update_inactive_users_to_active_users? or update_active_users_to_inactive_users? not sure what is it you are trying to do

Reply:
something probably like unpause_inactive_users or update_paused_users, maybe the second since you're not necessarily unpausing all inactive users, more like updating paused users if necessary